ray-operator/config/samples/ray-job.kueue-toy-sample.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   # The default value is "K8sJobMode", meaning RayJob will submit the Ray job via a submitter Kubernetes Job.
   # The alternative value is "HTTPMode", indicating that KubeRay will submit the Ray job by sending an HTTP request to the RayCluster.
   # submissionMode: "K8sJobMode"
-  entrypoint: python /home/ray/samples/sample_code.py
+  entrypoint: python -c "import time; [print(i) or time.sleep(1) for i in range(600)]"
```

Comment:
I like the example used here: https://docs.ray.io/en/master/cluster/kubernetes/getting-started/raycluster-quick-start.html#method-1-execute-a-ray-job-in-the-head-pod 

Can this be
```
entrypoint: python -c "import ray; ray.init(); print(ray.cluster_resources())"
```

I think an example that uses ray library will be better, even if it's very simple

Reply:
The `ray.cluster_resources()` example is not suitable for showcasing gang scheduling. In [this doc](https://docs.ray.io/en/master/cluster/kubernetes/k8s-ecosystem/kueue.html#kuberay-kueue), we create two RayJob custom resources and ensure that no Pods belonging to the second RayJob will be created because the ClusterQueue doesn't have enough resources. If the RayJob finishes very quickly, it is possible that users may not be able to observe this behavior.

Reply:
@andrewsykim does it make sense to you?

Reply:
Yes, that makes sense. This was more of a general comment that we don't need ConfigMap for most examples. [ray-job.sample.yaml](https://github.com/ray-project/kuberay/blob/master/ray-operator/config/samples/ray-job.sample.yaml#L10) for example can probably inline the example right?

Reply:
> The ray.cluster_resources() example is not suitable for showcasing gang scheduling.

Actually, even for this example, can we run `ray.cluster_resources()` in between each sleep?

Reply:
Updated https://github.com/ray-project/kuberay/pull/1976/commits/609343530180788ec6de8d7f64799f0d1d27b1cc